touchnet/models/kimi_audio/inference_kimi_audio.py

```
@@ -0,0 +1,153 @@
+                            sampler=sampler,
+                            shuffle=False,
+                            num_workers=args.num_workers,
+                            prefetch_factor=args.prefetch,
+                            collate_fn=collate_fn)
+
+    total_steps = len(dataset)
+
+    if rank == 0:
+        progress_bar = tqdm(total=total_steps, desc="Processing", unit="wavs")
+
+    writer = open(f"{args.output_dir}/part_{rank + 1}_of_{world_size}", "w")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)